taichi/perf.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include <taichi/common/util.h>
+#include <taichi/system/timer.h>
+
+TI_NAMESPACE_BEGIN
+
+#define TI_MAX_PERF_ID 32
+
+static inline double TI_PERF(const char *left = nullptr, int perf_id = 0, int show_freq = 1)
```

Comment:
Is there a reason for a new profiler?

Reply:
Sorry but I didn't realized there was an existing profiler..